queries/markdown/highlights.scm

```diff
@@ -19,6 +19,14 @@
   (fenced_code_block)
 ] @text.literal
 
+(pipe_table_header (pipe_table_cell) @text.title)
+[
+ (pipe_table_row)
+ (pipe_table_delimiter_row)
+ (pipe_table_header)
+] "|" @punctuation.special
```

Comment:
I think it might highlight ever "|" not just the ones in tables? Not sure though about this specific query syntax.

Reply:
I thought so first to, but it seems to work as expected when adding a `\|` in a cell. (with or without the inline)

Reply:
It works, because the only other way a `|` character can be in the text that should not be a row separator it has to be `\|`, but those are properly parsed by the markdown parser. That's why it works as intended

Reply:
If you checked it I'm happy :)